src/components/dashboard/accounting.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Header, Table, Segment } from 'semantic-ui-react';
+import moment from 'moment';
+
+/* Falta hacer el request para obtener los datos, el filtro por actividad y quizás paginar porque esta sección puede
+llegar a ser absurdamente larga */
+
+class DashboardAccounting extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isFetching: false,
+      accountingInfo: []
+    };
+  }
+  componentDidMount() {
+    this.setState({ isFetching: true });
```

Comment:
Es mejor que el isFetching sea true en el this.state, y luego se cambie una sola vez a "false" dps del didMount.. recuerdo que a mi para otro proyecto lo hice así como tu y me fallaba, pero no estoy tan seguro.

Reply:
Funciona